pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -455,6 +457,9 @@ func (br *OVSBridge) createTunnelPort(
 	if csum {
 		options["csum"] = "true"
 	}
+	if tunnelPort != 0 {
+		options["dst_port"] = strconv.Itoa(int(tunnelPort))
```

Comment:
Note that there are already some code specify dst_port in `extraOptions` and perhaps we should unify here. Since dst_port is just an option, perhaps do the same as other code: setting an option in the `extraOptions` map when user sets it.

Reply:
Thanks for the review, updated.